lib/config/baseline.js

```diff
@@ -6,16 +6,12 @@ module.exports = {
           autostart: false,
           max: 5,
           min: 1,
-          evictionRunIntervalMillis: 10000,
-          idleTimeoutMillis: 60000,
           testOnBorrow: true,
         },
         confirmPool: {
           autostart: false,
           max: 5,
           min: 1,
-          evictionRunIntervalMillis: 10000,
-          idleTimeoutMillis: 60000,
```

Comment:
The problem with removing the eviction / idle option, is that pools will only grow and never shrink (up to their max size). With a system that is configured for high peaks, this could substantially increase client and broker resources.

> Channels also consume resources in both client and server. Applications should minimize the number of channels they use when possible and close channels that are no longer necessary. Channels, like connections, are meant to be long lived.

https://www.rabbitmq.com/production-checklist.html#apps-excessive-channel-usage

Given this feature will require us to keep channels either

1. forever
1. until all reply handlers have been cancelled
1. until a configurable amount of time

Maybe we should use a separate channel pool. An alternative implementation would be to not return the channel until the replies have been cancelled. I've only just thought of this, but quite like it, as it would mean we could register the consumer on the channel when we borrow it, rather than on channel creation.